baseline/lexicon.yaml

```diff
@@ -94,10 +94,9 @@
     the primary deliverable in a release.
 - term: Collaborator
   definition: |
-    A user with a role on the project's version
-    control system who can approve changes or
-    manage the repository settings. Collaborators
-    may have varying permission levels based on
+    A human or non-human entity with permissions to
```

Comment:
Security tooling operating on the repository may have permissions to approve changes and manage repository settings.

I also expect many projects have change approval bots that may be permitted to operate without human change approval intervention.

Reply:
The only issue I see is in [OSPS-AC-01](https://baseline.openssf.org/versions/devel#osps-ac-01---the-projects-version-control-system-must-require-multi-factor-authentication-for-collaborators-modifying-the-project-repository-settings-or-accessing-sensitive-data):

It's not clear how 

> When a user attempts to access a sensitive resource in the project's version control system, the system MUST require the user to complete a multi-factor authentication process.

works with a non-human actor.

Reply:
I didn't make updates there because of this language:

> When a user attempts to access a [sensitive resource](https://baseline.openssf.org/versions/devel#sensitive-resource) in the project's [version control system](https://baseline.openssf.org/versions/devel#version-control-system), the system MUST require the user to complete a [multi-factor authentication](https://baseline.openssf.org/versions/devel#multi-factor-authentication) process.

"user" == human in my mind

Reply:
We could certainly speak to how non-human actors must authn/authz, but I didn't read the existing requirement as covering that.

Reply:
I also interpret "user" as explicitly "human", but we don't define that. Maybe we just change "collaborators" to "users" in the recommendation for AC-01? That way we're using consistent wording. Or else we can add a definition of user that makes it explicitly clear that it's a human actor.

Reply:
The intent of using "collaborator" in AC-01 stems from the intent that collaborator == human contributor with elevated permissions.

If we are changing collaborator to include non-human, then I agree that it should be clear that this is discussing human actors.

On an aside... The more we debate this, the more I question whether a non-human actor performing highly-sensitive actions should be tolerated as safe/secure.

Reply:
Does https://github.com/ossf/security-baseline/pull/304/commits/432faa8e28e4eacbe997353b170b1603d7315ec8 help?